### ISSUE TITLE
Suggestion : Add the possibility for any weapon to harvest

### ISSUE THREAD

**Issue by sawcisson** [suriyun-production/mmorpg-kit-docs#452]
Hi,

Would it be possible to have maybe a "checkbox" or an item that would allow any weapon type to harvest a harvestable ? Instead of listing all the weapons.

Or make it like for example : all melee items can harvest it 

I think it would be a great addition to the general workflow.

Thanks

**Comment by Gui205** [comment]
You can actually do that, you have to setup the harvest damage amount on your weapon. You are not limited on weapon type for your ressource so you can make all your weapon capable to harvest all your ressource with different effectiness for each weapon you want

**Comment by sawcisson** [comment]
The fact is that i would like only certain resources to be harvestables by all weapons.
I don't want a pickaxe to be able to harvest a tree for example but i would like a character to be able to harvest a small bush even if he has a pickaxe equipped.

**Comment by Gui205** [comment]
Yes you can do it, just add the axe in your ressource scripatble object, the only problem is he will play the axe attack animation attack when you harvest the ressource.

**Comment by sawcisson** [comment]
Yup that's what i meant i have to add each weapon for each harvestable.
And if i have a harvestable that's supposed to be used by any weapon type i have to add all the weapons to it and i can't just mark it as "harvestable by anything"

If the "Harvestable effectivenesses" of a harvestable item is set to none then nothing can harvest it.
